### PR TITLE
[counter]: support enable/disable counter query

### PR DESCRIPTION
**Pull request by sihuihan88**
Signed-off-by: Sihui Han <sihan@microsoft.com>

### CODE REVIEW

**Comment by sihuihan88** [comment]
Depends on https://github.com/Azure/sonic-swss-common/pull/202

**Comment by lguohan** [comment]
retest this please

File: syncd/syncd_flex_counter.h [hunk 1]
```diff
@@ -119,6 +123,7 @@ class FlexCounter
 
         uint32_t m_pollInterval = 0;
         std::string m_instanceId;
+        bool m_enable = true;
```
Comment (pavel-shirshov) [inline syncd/syncd_flex_counter.h]: Hi Sihui,

Can you please make it disabled by default?

**Comment by sihuihan88** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Pull request merged**